src/main/java/de/azapps/kafkabackup/common/topic/restore/RestoreTopicsArgsWrapper.java

```diff
@@ -0,0 +1,82 @@
+package de.azapps.kafkabackup.common.topic.restore;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Properties;
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@ToString
+public class RestoreTopicsArgsWrapper {
+
+  public static final String KAFKA_CONFIG_BACKUP_BUCKET = "aws.s3.bucketNameForConfig";
+  public static final String KAFKA_BOOTSTRAP_SERVERS = "kafka.bootstrapServers";
+
+  public static final String RESTORE_DRY_RUN = "restore.dryRun";
+  public static final String RESTORE_IGNORE_EXISTING_TOPICS = "restore.ignoreExistingTopics";
```

Comment:
What are the consequences of not ignoring? Will we error, or overwrite config, or?

Reply:
I think we should error with info that topic already exists.

I'm assuming either restoring to empty cluster or recreating some topics. This could be needed either because of sth being deleted or restore process did not go well and we want to reprocess one topic.

For second scenario restore tool would skip all existing topics and create only those that does not exist.
Alternatively we could add parameter for list of topics that we want to restore and try to restore only those.

Reply:
Actually after further discussion with @maciejmiedzianowski we had agreed that it would be better option to accept list of topic to recreate in second scenario.

It will align better with kafka API that accepts list of topics to be created and not single topic that we could skip if ignoring

Reply:
Implementation of restoring service will check if any of requested topics exists in cluster and of so will not restore configuration.